docs/getting-started.md

```diff
@@ -0,0 +1,80 @@
+# Prerequisites
+
+- [We recommend using VOLTA to manage your JavaScript command-line tools](volta.sh)
+- Node.js® version 14.18.1
+- Yarn - Please use Yarn v1 instead of npm to install dependencies  
+  `npm install --global yarn@1.22.17`
```

Comment:
suggestion: In order to minimize setup differences between machines, I think we should promote Volta more and say that's how work on UI should be done. So my suggestion is to remove the lines about Node.js and Yarn and ask people to install Volta. My understanding is that Volta will read the `package.json` file and figure out on its own which versions of Node.js and Yarn to install.